frontend/tests/index.html

```diff
@@ -27,6 +27,8 @@
     <script src="{{rootURL}}assets/frontend.js"></script>
     <script src="{{rootURL}}assets/tests.js"></script>
 
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```

Comment:
Für was brauchst du das genau?

Reply:
Damit die Bootstrap Komponenten in den Tests auch funktionieren, sonst knallts weil die function fehlt